apps/api/migrations/080_create_fraud_detection_tables.up.sql

```
@@ -0,0 +1,112 @@
+    avg_hourly_transactions FLOAT NOT NULL DEFAULT 0,
+    known_destination_count INT NOT NULL DEFAULT 0,
+    known_device_count INT NOT NULL DEFAULT 0,
+    typical_hour_start INT NOT NULL DEFAULT 0,
+    typical_hour_end INT NOT NULL DEFAULT 23,
+    transaction_count INT NOT NULL DEFAULT 0,
+    last_updated_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    UNIQUE(user_id)
+);
+
+CREATE INDEX IF NOT EXISTS idx_user_baselines_user_id ON user_baselines(user_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)